### DISCUSSION TITLE
Do you use the Simple App Launcher?

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#66] [Polls]
**This is about the app hosted at [Simple-App-Launcher](https://github.com/FossifyOrg/App-Launcher)**, not the [Simple-Launcher](https://github.com/FossifyOrg/Launcher).

If you do use it, please comment your reason below.

**Comment by inson1** [comment]
first launch always takes so much time (I cant use add apps button)

**Comment by inson1** [comment]
@naveensingh Do yout think of droping it?

**Reply by naveensingh** [reply]
@inson1 don't know yet, just looking for more data. This app has 1 million+ downloads on the Google Play Store, which means it does appeal to some people or maybe it's because of the simple brand.

**Reply by inson1** [reply]
@naveensingh yea I tried it, because of that, but deleted it after that.

Now it doesnt have so many stars, but in SMT   it has 4x than launcher and almost same as voice recorder and more than keyboard, draw, flashlight

But idk why would I should use it. I dont have any use case for that. I would find better launcher if I had this kind of problem.

**Reply by inson1** [reply]
@naveensingh I think its importnat to ask if there will be needed be any bigger changes? No? Why not keep it? But I wouldnt spend too much time on that tbh.

**Reply by Ponytronics** [reply]
I have a degoogled phone and enjoy the default black background of the home screen. So I don't use the launcher because I don't use any launcher ✌️

**Comment by Mr-Bajs** [comment]
No, never felt there where anything wrong with the AOSP launcher, so never felt the need to try anything else.

**Reply by Aga-C** [reply]
It's not about Simple Launcher, but Simple App Launcher - something like a drawer for the apps, not a real launcher.

**Reply by inson1** [reply]
But why not change launcher, basic launcher has "folders" and search. I dont see any good use case tbh.

**Reply by Mr-Bajs** [reply]
Sorry, i know. Should made that clear, i find that the "app drawer" are fine in AOSP. so never felt the need to install another one. But since i never tried it so can't tell. Though i remeber that Samung uses a proprietary app drawer that enables the user to hide apps in the drawer to clean it up, a feature that AOSP doesn't. I prefer to overall have as few apps as possible, and rather organising them into groups in the homescreen. But that just me.

**Comment by KenTheTester2** [comment]
I make heavy use of it as a shortcut menu honestly.  I use it on an ROG Phone 6 and have one of the side pressure buttons binded to bring up the app launcher. works  like a charm and I don't have to dig through my  more bloated traditional home screen and such. 

Also did something similar back when I was using both a umidigi phone briefly and samsung. I'd imagine most people used it in a similar manner. 

 I also find it the best way to launch apps in a different language than the rest of the phone  as I've had issues with normal home screen replacements and launchers when it comes to apps in different languages than what's set as default.

**Comment by keybreak** [comment]
I always recommend it for people who are not on DeGoogled ROMs yet, rather on some dodgy chinese phones...
It's definitely better than using their spyware launcher / app-launcher :joy:

**Reply by Ponytronics** [reply]
Yeah, those launchers that try and look like they are running on iOS!

**Comment by vagosss** [comment]
The only thing that is holding me back from using a 3rd party launcher on miui at least, is that it doesnt allow gesture navigation.

**Reply by Ponytronics** [reply]
In awe of you. I come from a planet where our webbed hands just can't work with gesture navigation just doesn't work for us. When I arrived on Earth, I obtained a degoogled Pixel 7a. Spent two months in misery before I realized I could switch to button navigation.

**Comment by Kreuger** [comment]
I use Neo Launcher.

**Comment by GitCurious** [comment]
If I wasn't already using Nova 7 I _might_ have used it - but at this time I don't.

**Reply by GuitarBilly** [reply]
i second this, also using Nova which has plenty of configurable options to suit my needs.

p.s. and I do not have a ton of apps installed.

**Comment by Mr-Bajs** [comment]
@naveensingh whatever you choose to do, it would i suppose be an good idea to change the name to App Drawer or something like that rather than App Launhcer to avoid any further confusion.

**Reply by ghost** [reply]
I just realized there's already a discussion dedicated to this in the App Launcher repo. See [here](https://github.com/FossifyOrg/App-Launcher/discussions/2).

**Comment by ghost** [comment]
Personally, I don't use it and I admit that for a while I had a hard time finding interest in having both the Launcher app and the App Launcher app in the suite.

As @KenTheTester2 did well notice, most people must use it as a shortcut drawer or a launcher substitute, all the more than the Launcher app was only launched in 2022 while the App Launcher was created in 2016. So for at least 6 years, the App Launcher made sense in the SMT suite.

Totally abandoning the App Launcher would of course be a shame for the people who use it, however I don't think it's necessary to invest a lot in this app.

Finally, I totally agree with @Mr-Bajs : "App Launcher" and "Launcher" are too similar and confusing. Renaming the App Launcher seems necessary to me, if you decide to continue to maintain it.

**Comment by KenTheTester2** [comment]
I'd like to add I don't think the app would need much investment,  given it already serves its general purpose well, Be it for people using as a secondary launcher by the sound of things or like how I use it as a shortcut menu. It would probably be enough to  just do bare minimum maintenance to avoid it being removed or hidden in the playstore given googles push to hide apps that don't get updated for a certain amount of time. (as well as block installs of said apps even when side loading.) that being said given I only have limited programming experience,  I'm unsure of how much work that actually is. If its too much I doubt too many would complain of its loss. Although I personally find the app very useful.  If you haven't already I'd suggest looking over the original apps reviews as alot of them are still very recent. 

ps: A name change to be less confusing is probably a good idea. Even I have gotten the 2 confused briefly when searching for it when setting up a new phone.

**Reply by ghost** [reply]
> It would probably be enough to just do bare minimum maintenance

I think the same thing. Doing only minimum maintenance (bug fixes for example, the app counts little but new ones could occur over time) is the best solution in my opinion. However, I’m afraid that it may be complicated to apply. Indeed, the fact that the app is part of a suite requires harmonisation. If the other apps in the suite were to evolve significantly, the App Launcher should therefore evolve as well.

I believe that the question of maintaining this app depends on the work it represents. I can't quantify it either, but the maintainer will know it and make the decision accordingly.

**Comment by friendlyflame** [comment]
I use it because it can hide apps, my stock launcher doesn't provide a way to hide apps. That's the only reason. And other launchers require ton of permissions.

**Comment by inson1** [comment]
https://www.reddit.com/r/Fossify/comments/18z8hj8/do_you_use_the_simple_app_launcher/

**Comment by LucaMkIII** [comment]
My use cases for the App Launcher have changed over time, but most have been related to the Launcher.

When I first was trying the Launcher, it didn't have folders (iirc), so I used the App Launcher to fill that need. (I was OK getting by with just one folder.) But I eventually missed some of the features* of my other launcher, and so I stopped using both the Launcher and the App Launcher.

More recently, I've been working back to using the Simple Launcher (to be replaced by the Fossify Launcher as soon as it's available). Since I'm tinkering and switching back and forth a lot between the two launchers, the App Launcher allows me to have a "folder" that exists in both launchers. However, that need will diminish as I use my old launcher less and less.

After that, *almost* everything the App Launcher does for me can be replaced by folders.

My only remaining use case is with apps for which I want to force a context-switch before using them, and the App Launcher does that better than home screen folders. But this is a niche use case, and I could make do with just folders if the App Launcher went away.

*Pin-able shortcuts! Also, folders in the app tray, and smoother visual transitions (a perk of being a built-in app). I'll probably feature-request some of these if I get around to it.

**Reply by inson1** [reply]
@LucaMkIII I guess stupid question, what is "context-switch" and why do you use it?

**Reply by LucaMkIII** [reply]
Not a stupid question at all.

If I open a folder on the home screen, I can still see the rest of the apps on my screen, so I'm still in the same "context": only part of my screen has changed, and I can smoothly transition from one mental task to another.

Sometimes, smooth transitions like this can help maintain "flow" state, but they also tend to keep me (personally) sucked into whatever I'm doing, even if I should take a break and work on something else.

When I open the App Launcher, the whole screen changes. I have no visual references to smooth the transition. So, the "context" is no longer maintained - it's a "context-switch".

This can create a jarring (or at least a distracting) user experience, pulling the user out of "flow"... And helping me remember I should get off my phone instead of opening that app I was looking for.

**Reply by LucaMkIII** [reply]
I actually found another use case - as a folder that hides what apps are inside it. When combined with the Launcher's "Hide app" feature, it allows those apps to be completely out of sight (on the home screen and app tray), but still easily accessible.

It could also act as a folder in the app tray (if you're OK with only one folder).

**Comment by TotallyAvailable** [comment]
Even while using a Launcher like Nova you might want to use something like Revanced (or build with different ID yourself) to get more App Launcher because why not.

Trying hard not to post another essay but yes, independent of more advanced Root related use cases, bigger collections of Apps (with whatever usage frequency), Tracking or Test setups/devices, App Launcher does provide quite a bit of simple yet attractive "freedom" to do whatever you want and feel like beyond the boundaries of your (freely chosen or not) launcher.

I am not going to bring up the topic of Icon Pack support again.
Neither am I going to ask you to provide more than 1 (packageid) version as GitHub release (maybe .fdroid for an eventual F-Droid release ?).
Just jumping in here in the hopes that "the absolute bare minimum" to keep it working within the Fossify family of Apps
isn't too much of a commitment or work vs the visible user base/provided feedback.

**Comment by p4bl0-** [comment]
I do use it daily. It's a nice way of having just my regularly used apps that cannot all fit on my home screen (of which half the space is used by a Notes widget with TODO items) in a single place and ordered as I see fits, while all other apps are still accessible in the messy all-apps drawers.

It just looks neat and clean and I like it.

![signal-2024-01-23-104728](https://github.com/FossifyOrg/General-Discussion/assets/8600047/fca372f1-7066-4ac0-acb5-1e82f6eeeefe)

**Comment by Danny3** [comment]
No!
I use Nova and I've been using it for many years.
I don't think any other launcher comes close to it.

**Reply by inson1** [reply]
btw app launcher is app drawer, not really launcher

**Reply by Danny3** [reply]
You mean Simple app launcher?
If it's an app drawer, can it be used with real app launchers?
Nova has its own app drawer and I like it a lot as it's very customizable, I can even set tabs and folders in it.

**Reply by inson1** [reply]
@Danny3 yes, yes, for example in this you could add only few selected apps, which yo want

**Comment by biopsin** [comment]
I would use the launcher if I could replace Trebuchet and still retain the gesture navigation, tried build [lawndesk](https://github.com/vipheyue/Lawndesk) with quickstep support but it still break taskswitch..

**Comment by ronniedroid** [comment]
@naveensingh I was just thinking about the app launcher/drawer today and I was thinking maybe we can either replace it with or turn it into a fossify-apps-updater.

This will be like obtainium but for simple apps. And I am planning to work on it if the community agrees. 

It will have the following features:

- List Fossify apps and make them launchable from there.
- Check for updates from github releases and notify the user about it.
- Show a change log for the update.
- Option to only check updates for installed apps or for all apps. (New release notification for not installed apps)


What do you think?

**Reply by naveensingh** [reply]
@ronniedroid Not a bad idea but I want to keep it as it is because those who use it like it because it is simple and also I don't think Google will allow an updater app on their store. 

Personally, I think we don't need an updater app because the existing solutions are already good enough. F-Droid, GPlay, and probably Obtainium too offer easy ways to update an app. You could always fork and add those Fossify Apps Updater features :)

**Reply by naveensingh** [reply]
And it'll need access to the Internet, not great.

**Reply by ronniedroid** [reply]
Yeah, google won't allow it, but I thought maybe would be good for ungoogled phones, but obtainium already does that. 

I'll scratch the idea then.

**Reply by naveensingh** [reply]
Thanks for trying anyway :)

We need ideas like this.

**Comment by n-ce** [comment]
### I don't use this app. But it's the closest FOSS app to my use case.
Even though I try to use FOSS apps where I can, I could not find any open source launcher that was just one single app grid.
But I found a closed source app that fits exactly my use case with some premium features. (Although the only premium feature that I required is the ability to hide apps.)
### [Zenit Launcher](https://play.google.com/store/apps/details?id=com.anrapps.zenit)
![Screenshot_20240128-165442](https://github.com/FossifyOrg/General-Discussion/assets/69345507/d314d563-c87d-47c1-96dd-a90c067345f7)

I would definitely use it if it was something like this. 

### Out of Topic
> I had been a **QKSMS** user for a long time but given it wasn't maintained in a long time so I had to switch to **Simple SMS** last year, but had to switch again (due to the sudden ads) to the newcomer **Deku SMS** (Fossify SMS wasn't ready by then) , few days ago it pushed an update that wasn't in my best interests, luckily **Fossify SMS** 1.0.0 was launched right then and I am happily using it right now, so thanks a lot! :)

**Reply by antodc** [reply]
@n-ce. I think you can achieve the same result with [Discreet Launcher](https://github.com/falzonv/discreet-launcher) by defining all apps as favorites (it takes 30 seconds).

**Reply by n-ce** [reply]
> @n-ce. I think you can achieve the same result with [Discreet Launcher](https://github.com/falzonv/discreet-launcher) by defining all apps as favorites (it takes 30 seconds).

Yes, you are right I used to use it for the same purpose 3 years ago. (You might see my name in the app as well)

Nowadays, I just use the stock launcher. :)
![Screenshot_20241024-184946](https://github.com/user-attachments/assets/aa40c5a7-be17-4122-a2f4-e07a62434c4e)

**Comment by emavgl** [comment]
Hi there, I want to share how I use the app. I have a Samsung Galaxy smartphone and they have a battery saving feature where you can access just a limited number of apps (if I remember correctly, 4). I use it mostly when I travel, to save the battery and arrive safe at the end of the day. Since the number of allowed apps is limited, I use the Simple App Launcher as one of the allowed apps, and by using that, accessing other apps.

**Comment by FerrusFox** [comment]
I use it to keep the simple apps together in the home screen on he simple launcher. I like it more than a folder, its more like a utilities screen.

**Comment by LynxAdventurerer** [comment]
Just learned of what happened to SMT tools and I love this project! The only app missing from Fossify at the moment that I used from SMT is the launcher. It was perfect for my needs.

**Reply by Cwpute** [reply]
what are your needs, and how was it different from other launchers ?

**Comment by antodc** [comment]
@naveensingh. Just out of curiosity. Have you taken any decision about whether to keep or drop the Launcher/Drawer app? Apologize if I missed the answer somewhere.

**Reply by GuitarBilly** [reply]
The launcher app is in beta testing since 19th October.

- announcement here   https://github.com/orgs/FossifyOrg/discussions/57
- development here   https://github.com/FossifyOrg/Launcher

If you like to try it then you can get it via F-Droid / IzzyOnDroid. Or sign up for beta testing for Google Play

**Reply by antodc** [reply]
@GuitarBilly. Excellent! Thanks a lot for the information.

**Reply by antodc** [reply]
Sorry. I actually meant "[App Launcher](https://github.com/FossifyOrg/App-Launcher)", not the app you pointed to.

**Reply by naveensingh** [reply]
Yeah, I have decided to drop it. The repository will likely be deleted, it's archived for now.

**Comment by ghost** [comment]
@naveensingh How about keeping Simple App Launcher alive as Fossify App Launcher in minimal maintenance mode? It looks like there were at least some users of it.

**Reply by naveensingh** [reply]
I don't think it makes sense as a separate app. The default home/launcher app should cover any use cases the App Launcher was designed for. I suggest organizing apps using folders or [tabs](https://github.com/FossifyOrg/Launcher/issues/46).

**Comment by Danny3** [comment]
No, because it has too many missing features compared to the Nova Launcher:
https://novalauncher.com/
That I've been using for more than a decade!
Mandatory features for me, that Nova provides are:
Multiple horizontal screen with an easy swipe left / right to switch between them.
Ability to put multiple apps in a folder with different displaying effects
Vertical drawer with search  bar and ability to group multiple apps in a folder or under a tab.
Naming / renaming of the folder or of an app.
I use the rename feature to also translate or describe what apps do in the native language of my parents.
